apps/laboratory/src/components/Ethers/EthersSignTypedDataTest.tsx

```diff
@@ -48,7 +48,12 @@ export function EthersSignTypedDataTest() {
 
       const signature = await signer?.signTypedData(domain, types, message)
 
-      toast({ title: 'Succcess', description: signature, status: 'success', isClosable: true })
+      toast({
+        title: 'Success',
+        description: `${signature.slice(0, 90)}...`,
```

Comment:
We are truncating the signature now? Could we add it to the page somewhere so we can copy the full signature? Useful for other things sometimes